src/feature/auto-closer.ts

```diff
@@ -94,6 +97,8 @@ export async function checkForAutoClose() {
         });
       }
 
+      await addLabels(client, issueMetadata, labels);
+
       await client.rest.issues.update({
```

Comment:
Could technically do it as part of the `update` calls, but I guess it abstracts the need to read the input better.

Reply:
Yeah, the `update` call will override the labels, so we'll have to read original ones from the input and append the new ones. Also might risk losing those added by moderators during processing time.